CONTRIBUTING.md

```diff
@@ -217,16 +210,6 @@ NPM_CONFIG_OTP={TOKEN} lerna publish --force-publish --conventional-graduate &&
 
 NOTE: With each prerelease the changelogs are getting updated. It might be wishful to delete the changelog before doing a final major release.
 
-#### Rate Limited OTP
-
-If publishing the packages fails with an error like this:
-
-```
-lerna ERR! E429 Could not authenticate ${npm-user-name}: rate limited otp
-```
-
-you simply need to wait five minutes before trying again. In case some packages have already been published and others have not, refer the advice about `lerna publish from-package` in the previous section.
-
```

Comment:
Discus: The error can still occur on the GH action I assume.

Reply:
I've not seen this issue in a long while, but we might as well keep this as long as we keep the other section.